source/upgrade/19.13/buster.rst

```diff
@@ -0,0 +1,88 @@
+.. _upgrade_notes_buster:
+
+********************************
+Debian 10 (buster) Upgrade Notes
+********************************
+
+The upgrade to Wazo 19.13 or later will take longer than usual, because the whole Debian system will
+be upgraded.
+
+The database management system (postgresql) will also be upgraded from version 9.6 to version 11 at
+the same time. This will upgrade the database used by Wazo. This operation should take at most a
+few minutes.
+
+After the upgrade, the system will need to be rebooted.
+
+
+Before the upgrade
+==================
+
+* Make sure your version of Wazo is at least 18.01. You can run ``wazo-upgrade`` to check the
+  version currently installed. If your version of Wazo is older that 18.01, you should first upgrade
+  your Wazo to Debian Stretch, following the procedure described in :ref:`upgrade-notes-stretch`.
+
+* Make sure your have sufficient space for the upgrade. You might run into trouble if you have less
+  than 2 GiB available in the file system that holds the :file:`/var` and :file:`/` directories.
+
+* If you have customized the Debian system of your Wazo in some nontrivial way, you might want to
+  review the `official Debian release notes <https://www.debian.org/releases/buster/releasenotes>`_
+  before the upgrade. Most importantly, you should:
+
+  * Make sure you don't have any unofficial sources in your :file:`/etc/apt/sources.list` or
+    :file:`/etc/apt/sources.list.d` directory. If you were using the ``stretch-backports`` source,
+    you must remove it.
+
+  * Remove packages that were automatically installed and are not needed anymore, by running
+    ``apt-get autoremove --purge``.
+
+  * Purge removed packages. You can see the list of packages in this state by running ``dpkg -l |
+    awk '/^rc/ { print $2 }'`` and purge all of them with ``apt-get purge $(dpkg -l | awk '/^rc/ { print $2 }')``
+
+  * Remove :file:`.dpkg-old`, :file:`.dpkg-dist` and :file:`.dpkg-new` files from previous upgrade.
+    You can see a list of these files by running ``find /etc -name '*.dpkg-old' -o -name '*.dpkg-dist' -o -name '*.dpkg-new'``.
+
+
+Upgrade
+=======
+
+The upgrade must be done with three commands:
+
+* ``wazo-dist -m pelican-stretch``: Ensures your system is not restricted to a specific version
+* ``wazo-upgrade``: Installs the ``wazo-dist-upgrade`` script and makes sure the system is up-to-date.
+* ``wazo-dist-upgrade``: Upgrade to the latest version of Wazo with Debian 9 (stretch). This upgrade
```

Comment:
Are these commands really going to upgrade a system to Buster? You mention 
> Upgrade to the latest version of Wazo with Debian 9 (stretch)

(also, the word Upgrade is missing a "s" at the end)

Reply:
Fixed, thanks :)